### PR TITLE
Misc: Change HTTP links to HTTPS where possible

### PR DESCRIPTION
**Pull request by mgol**
cc @AurelioDeRosa

### CODE REVIEW

**Comment by mgol** [comment]
If no one opposes I'd like to land it soon.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
+Welcome! Thanks for your interest in contributing to api.jquery.com. You're **almost** in the right place. More information on how to contribute to this and all other jQuery Foundation projects is over at [contribute.jquery.org](https://contribute.jquery.org). You'll definitely want to take a look at the articles on contributing [documentation](https://contribute.jquery.org/documentation).
 
-You may also want to take a look at our [commit & pull request guide](http://contribute.jquery.org/commits-and-pull-requests/) and [style guides](http://contribute.jquery.org/style-guide/) for instructions on how to maintain your fork and submit your code. Before we can merge any pull request, we'll also need you to sign our [contributor license agreement](http://contribute.jquery.org/cla).
+You may also want to take a look at our [commit & pull request guide](https://contribute.jquery.org/commits-and-pull-requests/) and [style guides](httsp://contribute.jquery.org/style-guide/) for instructions on how to maintain your fork and submit your code. Before we can merge any pull request, we'll also need you to sign our [contributor license agreement](https://contribute.jquery.org/cla).
```
Comment (dmethvin) [inline CONTRIBUTING.md]: Typo here: `httsp`

Comment (mgol) [reply CONTRIBUTING.md]: Good eye!

**Comment by mgol** [comment]
Thanks, Dave. Typo corrected & landed.

**Pull request merged**